spec/import_spec.rb

```diff
@@ -46,24 +46,24 @@ def upload_file!(name, ext='csv')
 
   context "posts index" do
     before do
-        Author.create!(name: 'John', last_name: 'Doe')
-        Author.create!(name: 'Jane', last_name: 'Roe')
-        add_post_resource({
-          validate: true,
-          headers_rewrites: {
-              :'Author Name' => :author_id
-          },
-          before_batch_import: ->(importer) do
-            authors_names = importer.values_at(:author_id)
-            # replacing author name with author id
-            authors   = Author.where(name: authors_names).pluck(:name, :id)
-            #{"Jane" => 2, "John" => 1}
-            options = Hash[*authors.flatten]
-            importer.batch_replace(:author_id, options)
-          end
-        })
-        visit '/admin/posts/import'
-        upload_file!(:posts)
+      Author.create!(name: "John", last_name: "Doe")
+      Author.create!(name: "Jane", last_name: "Roe")
+      add_post_resource({
+        validate: true,
+        headers_rewrites: {
+          :'Author Name' => :author_id
+        },
+        before_batch_import: ->(importer) do
+          authors_names = importer.values_at(:author_id)
+          # replacing author name with author id
+          authors   = Author.where(name: authors_names).pluck(:name, :id)
+          #{"Jane" => 2, "John" => 1}
+          options = Hash[*authors.flatten]
+          importer.batch_replace(:author_id, options)
+        end
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.